packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -125,6 +150,101 @@ function updateCurrentModel(model: UI5Model | undefined) {
   }
 }
 
+async function showWebView() {
+  const columnToShowIn =
+    window.activeTextEditor && window.activeTextEditor.viewColumn
+      ? window.activeTextEditor.viewColumn + 1
+      : ViewColumn.One;
+  const apiRefUrl = currentModel?.url;
+  if (!apiRefUrl)
+    throw Error("UI5-Language-Assistant: Model not initialised, try again");
+  const document = window.activeTextEditor?.document;
+  let name: string | undefined;
+  if (
+    currentModel?.framework &&
+    currentModel.version &&
+    window.activeTextEditor?.selection.active &&
+    document
+  ) {
+    name = await getNodeName(
+      document.getText(),
+      document.offsetAt(window.activeTextEditor?.selection.active),
+      currentModel.cachePath,
+      currentModel.framework,
+      currentModel.version
+    ).catch((Error) => {
+      console.log(Error);
+      return undefined;
+    });
+  }
+  const ui5Url = `${new URL(apiRefUrl).href}${name ? "#/api/" + name : ""}`;
```

Comment:
should not there be re-use in build the API ref link?
This is also done in the "more details" in hover feature...